include/supl_os_client.rst

```diff
@@ -0,0 +1,37 @@
+.. _supl_client:
+
+SUPL client
+###########
+
+The Secure User-Plane Location (SUPL) client module integrates the external SUPL client library into the |NCS|.
+This library can be used to receive A-GPS data over the SUPL protocol.
+
+The SUPL client library is released under a different license than the |NCS|.
+Therefore, you must download and install it separately to be able to use this module.
+Before you download the library, read through the license on the webpage.
+You must accept the license before you can download the library.
+
+Downloadine and installing
```

Comment:
Why is this downloadine again?

Reply:
I assume it's a typo and should be "Downloading"